hack/istio/skupper/install-skupper-demo.sh

```diff
@@ -447,9 +411,8 @@ openshift_install_skupper() {
   while ! ${CLIENT_EXE} -n ${MYSQLSKUPPERNS} get svc mysqldb-v1 &> /dev/null ; do echo -n '.'; sleep 1; done; echo
   SKUPPER_MYSQL_IP="$(${CLIENT_EXE} -n ${MYSQLSKUPPERNS} get svc mysqldb-v1 -o jsonpath='{.spec.clusterIPs[0]}')"
   infomsg "MySQL IP over the Skupper link: ${SKUPPER_MYSQL_IP}"
-  infomsg "Configuring Bookinfo ratings-v2-mysql to talk to MySQL over the Skupper link via the ServiceEntry"
-  ${CLIENT_EXE} -n bookinfo set env deployment/ratings-v2-mysql MYSQL_DB_HOST="${SKUPPER_MYSQL_IP}"
-  ${CLIENT_EXE} -n bookinfo patch serviceentry mysqldb.test --type='json' -p='[{"op": "replace", "path": "/spec/addresses/0", "value": "'${SKUPPER_MYSQL_IP}'"}, {"op": "replace", "path": "/spec/endpoints/0/address", "value": "'${SKUPPER_MYSQL_IP}'"}]'
+  infomsg "Configuring Bookinfo ratings-v2-mysql to talk to MySQL over the Skupper link"
+  ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n bookinfo set env deployment/ratings-v2-mysql MYSQL_DB_URL="mysqldb://${SKUPPER_MYSQL_IP}:3306/test"
```

Comment:
So, unless there is hidden support for MYSQL_DB_URL, I think this should be:

```suggestion
  ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n bookinfo set env deployment/ratings-v2-mysql MYSQL_DB_HOST="${SKUPPER_MYSQL_IP}"
```